### DISCUSSION TITLE
Please step-up as Crowdin admin

### DISCUSSION THREAD

**Discussion by ThomDietrich** [openhab/openhab-distro#1509] [openHAB Maintainers Discussions]
Hey all,
I hope you are all fine and safe in these difficult times.

I wanted to bring to your attention, that I am still the default contact for user reach-out on Crowdin. See: https://crowdin.com/profile/openhab-bot

![image](https://user-images.githubusercontent.com/2870104/104909229-d8f00e00-5987-11eb-80ac-fea4631bbc27.png)

All of these projects belong to the account "openhab-bot" which credentials only @kaikreuzer  knows. Beyond that I am a maintainer of all projects. I want to step down as this general maintainer and am looking for a replacement. If the following is not in general interest, please propose a different approach.

We are looking for a real person to do general maintenance tasks and to be the frontman for user requests. Typical tasks could be clean-up or the adding of new repositories; typical requests are "I want to translate openHAB to Slovenian, please add the language to all projects."

Any further thoughts by you guys? Are You interested? Are there any especially active translators on crowdin we could reach out to? Please discuss :)

**Comment by comradekingu** [comment]
@ThomDietrich 
Happy to help if you use Weblate, and in setting that up. :)
